src/app/src/main/kotlin/org/imperial/mrc/hint/controllers/ProjectsController.kt

```diff
@@ -80,9 +80,9 @@ class ProjectsController(private val session: Session,
         @RequestParam("name") name: String,
         @RequestParam("note") note: String?): ResponseEntity<String>
     {
-        val newProjectId = projectRepository.saveNewProject(userId(), name, note = note)
+        val newProjectId = projectRepository.saveNewProject(userId(), name)
```

Comment:
So, just to confirm: when creating a new project from an existing project version then the note attached to the version should be transferred to the first version of the new project. And the new project itself won't have its note populated, either from the original project or version?

Reply:
That's right. The project itself won't have its note populated but the version.